packages/docusaurus-types/src/index.d.ts

```diff
@@ -190,13 +190,6 @@ export interface Props extends LoadContext, InjectedHtmlTags {
   plugins: Plugin<unknown>[];
 }
 
-/**
- * Same as `Props` but also has webpack stats appended.
- */
-export interface PropsPostBuild extends Props {
-  stats: Stats.ToJsonOutput;
```

Comment:
It was getting quite confusing, since webpack doesn't really document a lot of the specifics.  Plus I found a way around it.  I'm happy to add it back if you want.

Reply:
We can remove it for now.  If any user was using it and complains we'll add it back